src/activity_handlers.cpp

```diff
@@ -1588,7 +1588,7 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
         const int original_charges = liquid.charges;
         if( liquid.has_temperature() && liquid.specific_energy < 0 ) {
             liquid.set_item_temperature( std::max( temp_to_kelvin( get_weather().get_temperature( p->pos() ) ),
-                                                   277.15 ) );
+                                                   static_cast<double>( temperatures::cold ) ) );
```

Comment:
Isn't `temperatures::cold` in Celsius?

Reply:
It is in farenheit. But that temp_to_kelvin needs to move now since the map temperature and cold temperature are in farenheit but set_item_temperature wants kelvins.